kik_unofficial/client.py

```diff
@@ -466,16 +494,33 @@ def set_background_picture(self, filename):
         """
         self.log.info(f"Setting the background picture to filename '{filename}'")
         profile_pictures.set_background_picture(
-            filename, f'{self.kik_node}@talk.kik.com', self.username, self.password
-        )
+            filename, f'{self.kik_node}@talk.kik.com', self.username, self.password)
+
+    def set_group_picture(self, filename, group_jid: str, silent: bool = False):
```

Comment:
Can you add typings for `filename` parameter if it can also be `bytes`?

Reply:
I did this but I see an issue with the logging in these methods:

self.log.info(f"Setting the background picture to filename '{filename}'")

Won't this log a bunch of gibberish when raw bytes are passed in? What's the best way to resolve this when it's being logged?

Reply:
Yes, this is why I think it's better to separate these parameters, it's also misleading to call it `filename` when it can also contain other things

Reply:
9fe6eb08a506a3426bfcd8653ad217898f6f7397 should resolve this